src/views/CTLViewsHoC.jsx

```diff
@@ -151,6 +155,8 @@ const CTLViewsHoC = (WrappedView) => {
 										};
 									}
 									case 'trial-type': {
+										// const ttPayload = getListingInfo.payload[idx];
```

Comment:
same here  - probably you don't need this commented out code

Reply:
Also removed!